### PR TITLE
Make image prefix configurable

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

**Comment by niladrih** [comment]
bors try

**Review by avishnu — APPROVED** [review]
LGTM

File: default.nix [hunk 1]
```diff
@@ -1,9 +1,9 @@
-{ system ? null, allInOne ? true, incremental ? false, img_tag ? "", tag ? "", img_org ? "" }:
+{ system ? null, allInOne ? true, incremental ? false, img_tag ? "", tag ? "", img_org ? "", product_prefix ? "" }:
```
Comment (tiagolobocastro) [inline default.nix:1]: We already have img_tag and img_org, so let's use img_prefix here too

File: Jenkinsfile [hunk 2]
```diff
@@ -85,6 +85,7 @@ pipeline {
   }
   environment {
     CARGO_INCREMENTAL = "0"
+    PRODUCT_PREFIX = 'mayastor'
```
Comment (tiagolobocastro) [inline Jenkinsfile]: Maybe then here should also be IMAGE_PREFIX ?

Comment (niladrih) [reply Jenkinsfile]: Eventually this value would we used with the kubectl-mayastor binary to decide on the suffix name.

Comment (niladrih) [reply Jenkinsfile]: Does IMAGE_PREFIX still fit?

Comment (tiagolobocastro) [reply Jenkinsfile]: hmm good point, maybe leave this one as is then

File: default.nix (hunk 1 above)
Comment (niladrih) [reply default.nix:1]: Yup, makes sense. Changed the name of the variable.

**Comment by niladrih** [comment]
bors try

**Review by tiagolobocastro — APPROVED** [review]

**Comment by niladrih** [comment]
bors merge

**Pull request merged**